sormas-backend/src/main/java/de/symeda/sormas/backend/sormastosormas/share/ShareDataBuilder.java

```diff
@@ -51,6 +64,8 @@ public SHARED buildShareData(ADO data, ShareRequestInfo requestInfo) throws Sorm
 		return shared;
 	}
 
+	protected abstract void validateWithEjbShared(SHARED shared) throws ValidationRuntimeException;
```

Comment:
I think `doBusinessValidation` would be a better name

Reply:
updated